### PR TITLE
Fixrad

### PR DESCRIPTION
**Pull request by mbellew**
Minor optimization to PerPixelMath

Tracked down a rendering discrepancy to the different scaling of "rad" variable.
  This affected some presets that are very sensitive to zoom/zoomexp such as  "Aderrasi - Contortion (Escher's Tunnel Mix).milk"
  see also "Idiot24-7 - Ascending to heaven 2.milk"

### CODE REVIEW

File: src/libprojectM/MilkdropPresetFactory/PresetFrameIO.cpp [hunk 1]
```diff
@@ -179,59 +179,52 @@ void PresetOutputs::Render(const BeatDetect &music, const PipelineContext &conte
...
+            {
+                const float cos_rot = cosf(this->rot_mesh[x][y]);
+                const float sin_rot = sinf(this->rot_mesh[x][y]);
+                const float u2 = u - this->cx_mesh[x][y];
+                const float v2 = v - this->cy_mesh[x][y];
+                u = u2 * cos_rot - v2 * sin_rot + this->cx_mesh[x][y];
+                v = u2 * sin_rot + v2 * cos_rot + this->cy_mesh[x][y];
+            }
+            this->x_mesh[x][y] = u - this->dx_mesh[x][y];
+            this->y_mesh[x][y] = v  - this->dy_mesh[x][y];
```
Comment (revmischa) [inline src/libprojectM/MilkdropPresetFactory/PresetFrameIO.cpp:227]:
```suggestion
            this->y_mesh[x][y] = v - this->dy_mesh[x][y];
```

**Review by revmischa — APPROVED** [review]
Cool I'll assume this is better lol

**Comment by mbellew** [comment]
Man that diff looks a lot worse than it is.  Mostly just a) one loop instead of several b) skipping computation that don't do anything e.g.  when zoom=1 or rot=0.

**Comment by revmischa** [comment]
yeah we seriously need to run some formatting tool on everything

**Pull request merged**